website/src/components/Changelog/Android.tsx

```diff
@@ -19,15 +19,16 @@ export default function Android() {
   return (
     <Entries downloadLinks={downloadLinks} title="Android">
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-      <Unreleased>
+      <Unreleased></Unreleased>
+      <Entry version="1.4.6" date={new Date("2025-04-14")}>
```

Comment:
```suggestion
      <Entry version="1.4.6" date={new Date("2025-04-15")}>
```

In UTC, it is technically already 15/04 :)